index.html

```diff
@@ -0,0 +1,87 @@
+<html>
+
+<head>
+  <title>SQL tests</title>
+  <script src="packages/auth/dist/auth.umd.js"></script>
+  <script src="packages/sql/dist/sql.umd.js"></script>
+</head>
+
+<body>
+  <div id="online">
+    <button id="logoutBtn">Logout</button>
+    <button id="sqlBtn">Truncate, copy 1e6 rows and select 1000</button>
+  </div>
+  <div id="offline">
+    <button id="loginBtn">Login</button>
+  </div>
+  <script>
+    // Who needs react
+    function render() {
+      if (oauth.expired) {
+        online.style.display = 'none';
+        offline.style.display = 'block';
+      } else {
+        online.style.display = 'block';
+        offline.style.display = 'none';
+      }
+    }
+
+    async function init(token) {
+      const sql = new cartoToolkitSql.SQL(
+        'roman-carto',
+        oauth.token
+      );
+
+      function getFile(rows = 7) {
+        const contents = ['id,name'];
+
+        for (var i = 0; i < rows; i++) {
+          contents.push(`${i},A`);
+        }
+
+        return contents.join('\n');
+      }
+
+      async function copyFile(rows = 7) {
+        await sql.query('truncate copy_test');
+
+        const file = getFile(rows);
+
+        const result = await sql.copyFrom(file, 'copy_test', ['id', 'name']);
+
+        console.log(result);
+
+        const data = await sql.query('SELECT * FROM copy_test LIMIT 100');
+
+        return data;
+      }
+
+      sqlBtn.addEventListener('click', async () => {
+        const data = await copyFile(1e6);
+
+        console.log(data);
+      });
+    }
+
+    const oauth = new cartoToolkitAuth.OAuth({
+      clientID: 'fmka4wCAclcn',
+      redirectURI: 'https://localhost:5000',
```

Comment:
How does anyone set this local https for a quick test? Some notes in the corresponding README?

Reply:
There's a few examples around. Really, what you have to do, is figure out how to generate a cert/pem/key on you OS, then you can use [`http-server`](https://www.npmjs.com/package/http-server) or any other.

It's kind of annoything, react-cli and vue-cli have http servers that generate a self signed certificate for you, but it's not on a ready to use package.

Zeit's serve used to do this, but it no longer has that feature